HW2/index.css

```diff
@@ -0,0 +1,260 @@
+	margin-right: 15px;
+	margin-left: 10px;
+	text-shadow: 1px 1px 1px rgba(0, 0, 0, 0.5);
+}
+.menu__link:first-child {
+	margin-left: 0px;
+}
+.menu__link:hover {
+	background-color: #353535;
+	border-right: 1px solid #838383;
+	border-left: 1px solid #838383;
+	margin-right: 14px;
+	margin-left: 9px;
+}
+.menu__link:first-child:hover {
+	margin-left: -1px;
+}
+
+/****************CONTENT***********/
+.container {
+	width: 100%;
+	display: flex;
+	justify-content: space-between;
+	margin-top: 40px;
+	background-color: #ffffff; 
+}
+.aside {
+	width: 25%;
+}
+.article {
+	width: 70%;
+}
+.aside li:last-child {
+	border-bottom: 1px solid #c8c8c8;
+}
+.aside li{
+	list-style: none;
+	border-top: 1px solid #c8c8c8;
+	border-left: 1px solid #c8c8c8;
+	border-right: 1px solid #c8c8c8;
+	font-family: "2211.ttf";
+}
+.aside__link {
+	font-size: 15px;
+	text-decoration: none;
+	color: #656363;
+	height: 40px;
+	padding: 10px 24px;
+	display: block;			
+}
+.aside__link:hover {
+	color: #2c2c2c;
+	background-color: #f9f9f9;
+	padding-left: 40px;
+
+}
+.article_picture {
+	border: 1px solid #c8c8c8;
+	width: 100%;
+	display: flex;
+	width: 694px;
+	height: 400px;
+}
+.photo {
+	margin: auto;
+}
+.description {
+	padding: 20px;
+	margin-top: 15px;
+	font-size: 15px;
+	color: #2c2c2c;
+	font-family: "2211.ttf";
+	line-height: 21px;
+}
+
+/**************FOOTER************/
+.footer {
+	margin-top: 35px;
+	margin-bottom: 10px;
+	background-color: #e1e1e1;
+	justify-content: space-between;
+	display: flex;
+}
+.footer__list{
+	background-color: #e1e1e1;
+	background-color: #ffffff;
+	background-image: linear-gradient(to top, #e1e1e1 0%, #e1e1e1 100%);
+}
+.footer__list li{
+	list-style: none;
+	margin-right: 15px;
+	margin-left: 10px;
+	display: inline-block;
+	font-family: "2211.ttf";
+}
+.footer__link {
+	text-decoration: none;
+	color: #444444;
+	font-size: 15px;
+	display: block;
+	height: 40px;	
+	padding-top: 10px;
+	padding-left: 15px;
+	padding-right: 15px;
+}
+.footer__link:hover {
+	text-decoration: underline;
+}
+.copyright {
+	color: #444444;
+	padding: 10px;
+}
+
+.float__footer{
+	float: left;
+}
+
+
+
+
+/**************FLOAT************/
+.header--float {
+	margin-top: 20px;
+}
+.menu--float {
+	background-color: #535252;
+	list-style: none;
+	background-color: #ffffff;
+    background-image: linear-gradient(to top, #444343 0%, #5e5e5e 100%);
+}
+.menu__item--float {
+	margin-right: 15px;
+	margin-left: 10px;
+	float: left;
+	height: 40px;
+	color: #fff;
+	padding: 10px 16px;
+	color: #fff;
+	font-size: 15px;
+}
+.menu--float:after {
+	content: "";
+	display: block;
+	height: 0;
+	clear: both;
+	visibility: hidden;
+}
+.menu__item--float:hover {
+	background-color: #353535;
+	border-right: 1px solid #838383;
+	border-left: 1px solid #838383;
+	margin-right: 14px;
+	margin-left: 9px;
+
+}
+.menu_item--float:first-child{
+	margin-left: 0px;
+}
+.menu__item--float:first-child:hover{
+	margin-left: -1px;
+
+}
+#product {
+margin-left: 13px;
+}
+#our {
```

Comment:
moreover don't name elements base on their text, cause it could be different anytime